nixos/modules/config/users-groups.nix

```diff
@@ -329,8 +333,10 @@ let
           {command}`passwd` command. Otherwise, it's
           equivalent to setting the {option}`hashedPassword` option.
 
-          Note that the {option}`hashedPassword` option will override
-          this option if both are set.
+          Note that, when {option}`users.mutableUsers` is true, the {option}`hashedPassword`
+          option will override both the {option}`user.initialHashedPassword` option and
+          `{option}`user.initialPassword` if set. One exception to this rule is if
+          [](#opt-systemd.sysusers.enable) is true.
```

Comment:
What is the exception?
Perhaps editing to something like: "... is true, in which case only `initialPassword`, `initialHashedPassword`, and `hashedPasswordFile` options will be used."

This also applies to Note one line 360